packages/api-client/src/components/CommandPalette/CommandPaletteExample.vue

```diff
@@ -7,24 +7,54 @@ import {
   ScalarDropdownItem,
   ScalarIcon,
 } from '@scalar/components'
+import type { Request } from '@scalar/oas-utils/entities/workspace/spec'
 import { onMounted, ref } from 'vue'
+import { useRouter } from 'vue-router'
+
+const props = defineProps<{
+  /** The request uid to pre-select */
+  metaData?: string
+}>()
 
 const emits = defineEmits<{
   (event: 'close'): void
 }>()
 
-const { activeWorkspaceRequests } = useWorkspace()
+const { push } = useRouter()
+const {
+  activeRequest,
+  activeWorkspace,
+  activeWorkspaceRequests,
+  requests,
+  requestExampleMutators,
+} = useWorkspace()
+
 const exampleName = ref('')
-const selectedRequest = ref(activeWorkspaceRequests.value[0])
+const selectedRequest = ref(
+  // Ensure we pre-select the correct request
+  requests[props.metaData ?? ''] ?? activeRequest.value,
```

Comment:
`requests[props.metaData ?? '']` wont this error? ohhh but then it defaults

Reply:
Up this bit allows us to delete all requests. We can probably add an empty state later

Reply:
Yep exactly. It will default